integration-cli/docker_cli_commit_test.go

```diff
@@ -83,9 +80,7 @@ func (s *DockerSuite) TestCommitHardlink(c *check.C) {
 			break
 		}
 	}
-	if !found {
-		c.Fatalf("Failed to create hardlink in a container. Expected to find %q in %q", inode, chunks[1:])
-	}
+	c.Assert(found, checker.True, check.Commentf("Failed to create hardlink in a container. Expected to find %q in %q", inode, chunks[1:]))
```

Comment:
I'm a little suspect of this test, and the test beneath.
I think we can simplify these.
Instead of doing the ranges I think we can do `c.Assert(chunks[1:], checker.Contains, inode)`